napari/_qt/layers/qt_labels_layer.py

```diff
@@ -18,6 +18,48 @@
 
 
 class QtLabelsControls(QtLayerControls):
+    """Qt view and controls for the napari Labels layer.
+
+    Parameters
+    ----------
+    layer : napari.layers.Layer
+        An instance of a napari layer.
+
+    Attributes
+    ----------
+    button_group : qtpy.QtWidgets.QButtonGroup
+        Button group of label layer controls: PAN_ZOOM, PICKER, PAINT, or FILL.
+    colormapUpdate : qtpy.QtWidgets.QPushButton
+        Button to update colormap of label layer.
+    contigCheckBox : qtpy.QtWidgets.QCheckBox
+        Checkbox to control if label layer is contiguous.
+    fill_button : qtpy.QtWidgets.QtModeRadioButton
+        Button to select FILL mode.
+        Available mode options are: PAN_ZOOM, PICKER, PAINT, or FILL
```

Comment:
Not sure it is necessary to list all the available modes here, fine to have them once in `button_group`, but not needed again for `fill_button`, `paint_button`, or `panzoom_button` i think